packages/jss/src/index.js

```diff
@@ -7,8 +7,27 @@
  * @license MIT
  */
 import Jss from './Jss'
+import StyleSheet from './StyleSheet'
 import type {JssOptions} from './types'
 
+/**
+ * Export types for better typing inside plugins and integrations.
+ */
+export type {
+  StyleSheetOptions,
+  JssValue,
+  JssOptions,
+  JssStyle,
+  Plugin,
+  generateClassName,
+  RuleListOptions,
+  Rule,
+  Renderer,
```

Comment:
Is this one needed?

Reply:
That depends on wether or not we allow people to write there own renderers?